.github/workflows/release.yml

```
@@ -117,6 +117,36 @@ jobs:
           echo "=== Current working directory ==="
           pwd
 
+      - name: Update the Homebrew tap
+        # Skipped automatically when the tap token is not configured, so the
+        # release still succeeds on a fork or before the tap exists.
+        if: ${{ secrets.HOMEBREW_TAP_TOKEN != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)